fluentd/plugins.sls

```diff
@@ -0,0 +1,12 @@
+{% if salt.pillar.get('fluentd:plugin_dependencies') %}
+install_fluentd_plugin_dependencies:
+  pkg.installed:
+    - pkgs: {{ salt.pillar.get('fluentd:plugin_dependencies') }}
+    - refresh: True
+    - require_in:
+        - gem: install_fluentd_plugins
+{% endif %}
+
+install_fluentd_plugins:
+  gem.installed:
+    - names: {{ salt.pillar.get('fluentd:plugins') }}
```

Comment:
Just so I know I'm looking at this right: This corresponds to https://github.com/mitodl/fluentd-formula/pull/1/files#diff-aef06cd118a72975a36aa9aeb06e8af6R3 if I were to rename and use the example pillar, right?